data_override/README.MD

```diff
@@ -10,26 +10,25 @@
 
 #### 1. YAML Data Table format:
 Each entry in the data_table has the following key values:
-- **gridname:** Name of the grid to interpolate the data to. The acceptable values are "ICE", "OCN", "ATM", and "LND"
-- **fieldname_code:** Name of the field as it is in the code to interpolate.
-- **fieldname_file:** Name of the field as it is writen in the file. **Required** only if overriding from a file
-- **file_name:** Name of the file where the variable is located, including the directory. **Required** only if overriding from a file
-- **interpol_method:** Method used to interpolate the field. The acceptable values are "bilinear", "bicubic", and "none". "none" implies that the field in the file is already in the model grid. The LIMA format is no longer supported. **Required** only if overriding from a file
+- **grid_name:** Name of the grid to interpolate the data to. The acceptable values are "ICE", "OCN", "ATM", and "LND"
+- **fieldname_in_model:** Name of the field as it is in the code to interpolate.
+- **override_file:** The parent key containing the nested keys related to overriding data from file. **Required** only if overriding from a file
+  - **file_name:** Name of the file where the variable is located, including the directory
+  - **fieldname_in_file:** Name of the field as it is writen in the file
+  - **interp_method:** Method used to interpolate the field. The acceptable values are "bilinear", "bicubic", and "none". "none" implies that the field in the file is already in the model grid. The LIMA format is no longer supported
+  - **multi_file:** The multifile parent key. **Required** only if it is desired to use multiple(3) input netcdf files instead of 1. Note that **file_name** must be the second file in the set when using multiple input netcdf files
+    - **prev_file_name:** The name of the first file in the set
+    - **next_file_name:** The name of the third file in the set
+  - **external_weights:** The external weights parent key. **Required** only if it is desired to use external weights from file
+    - **file_name:** Name of the file where the external weights are located, including the directory
+    - **source:** Name of the source that generated the external weights. The acceptable values are "esmf" and "fregrid"
```

Comment:
Are we supporting ESMF weight files?  I had put in PR#1399 that we were only supporting fregrid-generated.

Reply:
I updated the documentation